### PR TITLE
Tweak backbone modal

### PR DESCRIPTION
**Pull request by shivapoudel**
CC @jameskoster 
- Use of `border-box` and `content_h` math logic to fix the bug with radio buttons on tax table.
- Slightly decrease the header `h1` font size: `22px to 18px`
- Fix - Unusual slightly dark box on the backdrop of modal right side :octocat:  
- Updated with modal close button instead of link. Similar to `themes.php` modal close button ;)

### CODE REVIEW

**Comment by jameskoster** [comment]
More info please, what is going on here, and more importantly, why?

**Comment by shivapoudel** [comment]
@jameskoster the radio button for the tax table is not well displayed. So I have shifted to `box-sizing: border-box` and thus this requires the `content_h` math logic and some css style rules to fix the footer of backbone modal. Finally I have change the h1 font-size and of modal title and close button according to `themes.php`. See the difference via the screenshot below ;)

**Before**
previous

**After**
future

File: assets/css/admin.scss [hunk 1]
```diff
@@ -4144,17 +4135,20 @@ table.bar_chart {
...
 
-	footer .inner {
-		padding: 0 10px;
-		text-align: right;
+		.inner {
+			float: right;
+			line-height: 23px;
+
+			.button {
+				margin-bottom: 0;
```
Comment (shivapoudel) [inline assets/css/admin.scss]: This is to fix the height of footer area on responsive device xD

**Comment by jameskoster** [comment]
Thanks @shivapoudel happy with this. Will merge.

**Pull request merged**